packages/@o3r-training/tools/package.json

```diff
@@ -0,0 +1,136 @@
+{
+  "name": "@o3r-training/tools",
```

Comment:
why not using `@o3r` prefix ?

Reply:
we wanted to avoid invading o3r with private packages, but if you find the o3r prefix to be a better fit we can further discuss, what does everyone think ?

Reply:
I personally think that it's not necessary to use a separate scope.

If we do however, we need to make sure to update all the places when we rely on it
I am mainly thinking about npm registries for Verdaccio (which is currently breaking the it-tests on this PR) but there may be other things

Reply:
Good point, I will switch the scope of this package and will open an issue to facilitate the creation of new scopes within our repository (which will allow us to eventually migrate this package to the new scope as initially planned)

Reply:
https://github.com/AmadeusITGroup/otter/issues/2144